_pages/oidc/user-info.md

```diff
@@ -0,0 +1,98 @@
+---
+title: OpenID Connect
+lead: >
+    [OpenID Connect](http://openid.net){:class="usa-link--external"} is a simple identity layer built on top of the OAuth 2.0 protocol. Login.gov supports [version 1.0](http://openid.net/specs/openid-connect-core-1_0.html){:class="usa-link--external"} of the specification and conforms to the [iGov Profile](https://openid.net/wg/igov){:class="usa-link--external"}.
+sidenav:
+  - text: Getting started
+    href: "oidc/#getting-started"
+  - text: Authorization
+    href: "oidc/authorization/"
+  - text: Token
+    href: "oidc/token/"
+  - text: User info
+    href: "oidc/user-info/"
+    links:
+      - text: User info response
+        href: "#user-info-response"
+  - text: Certificates
+    href: "oidc/#certificates"
+  - text: Logout
+    href: "oidc/#logout"
+  - text: Example application
+    href: "oidc/#example-application"
+
+---
+{% capture front_matter %}
+## User info
+
+The user info endpoint is used to retrieve [user attributes]({{ site.baseurl }}/attributes/). Clients use the `access_token` from the [token response]({{ site.baseurl }}/oidc/token/#token-response) as a bearer token in the [HTTP Authorization header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization){:class="usa-link--external"}. To request attributes, send an HTTP GET request to the `/api/openid_connect/userinfo` endpoint, for example:
+
+### User info response
+
+The user info response will be a JSON object containing [user attributes]({{ site.baseurl }}/attributes/). Login.gov supports some of the [standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims){:class="usa-link--external"} from OpenID Connect 1.0. In addition to the user attributes, the following information will also be present:
+{% endcapture %}
+
+<div class="grid-row grid-gap">
+  <div class="desktop:grid-col-8 mobile:grid-col-full">
+    {{ front_matter | markdownify }}
+        <div class="dev-doc-row grid-row">
+            <div class="grid-col-5">
+                <h4 class="parameters">iss <span class="text-normal">(string)</span></h4>
+            </div>
+            <div class="grid-col-7">
+                The issuer of the response, which will be the URL of the Login.gov IdP, for example: <code class="language-plaintext highlighter-rouge">https://idp.int.identitysandbox.gov</code>
+                <ul>
+                    <li>Requires <code class="language-plaintext highlighter-rouge">profile</code> or <code class="language-plaintext highlighter-rouge">profile:name</code> scopes.</li>
```

Comment:
I recommend removing the period at the end of this sentence for consistency

Reply:
removed!